src/rpc.hpp

```diff
@@ -28,20 +33,42 @@ class ObjectSchema;
 namespace rpc {
 
 using json = nlohmann::json;
+
 using RPCObjectID = u_int64_t;
 using RPCRequest = std::function<json(const json)>;
 
+class RPCWorker {
```

Comment:
I tried to comment plenty inside the implementation file. You think I should do more here?